spec/05-classes-and-objects.md

```diff
@@ -435,13 +435,13 @@ An early definition is type-checked and evaluated in the scope which
 is in effect just before the template being defined, augmented by any
 type parameters of the enclosing class and by any early definitions
 preceding the one being defined. In particular, any reference to
-`this` in the right-hand side of an early definition refers
+`this` on the right-hand side of an early definition refers
 to the identity of `this` just outside the template. Consequently, it
 is impossible that an early definition refers to the object being
 constructed by the template, or refers to one of its fields and
 methods, except for any other preceding early definition in the same
 section. Furthermore, references to preceding early definitions
-always refer to the value that's defined there, and do not take into account
+always refer to the value that's defined there and do not take into account
```

Comment:
This comma serves the same purpose as the comma before "or" in the preceding sentence.

Reply:
I approved this because I thought the comma was incorrect because it doesn't precede an independent clause.

Maybe it's better to dodge the issue by starting a new sentence: "...defined there. They do not take..."

Reply:
We need a semicolon patrol like they do for tailrec and -Xlint.